go.sum

```diff
@@ -59,8 +59,8 @@ github.com/cpuguy83/go-md2man v1.0.10/go.mod h1:SmD6nW6nTyfqj6ABTjUi3V3JVMnlJmwc
 github.com/crossplaneio/crossplane v0.5.0-rc.0.20191211203849-05517d46525d h1:Xie+cgdanhW6PN1mbcMWYs7q//HM1z1O/JgzgQj0R5k=
 github.com/crossplaneio/crossplane v0.5.0-rc.0.20191211203849-05517d46525d/go.mod h1:f/e99LDWClmhkvMuvK7UMP1g5b718DGmdnpdrflK0HI=
 github.com/crossplaneio/crossplane-runtime v0.2.4-0.20191211004842-fa83d075257c/go.mod h1:aVLQmXiPeb/jhGGQFIxx4Rc1CFC6xq2BA6A3neLZIHA=
```

Comment:
similar to https://github.com/crossplaneio/stack-azure/pull/85/files#r362299283:

there are still refs to crossplane-runtime v0.2.4. Should we do something similar to what we did in https://github.com/crossplaneio/stack-rook/pull/32/files#r361492505, or just treat this as a simple backport only that does not introduce additional changes to the release branch outside of the commit being backported?